src/main/kotlin/org/partiql/scribe/targets/spark/SparkExcludeTranspilation.md

```diff
@@ -0,0 +1,105 @@
+This document goes over one of the more complicated rewrites for PartiQL's `EXCLUDE` to SparkSQL.
```

Comment:
self-review: would recommend reviewing this file before diving into the other files.